face/src/main/java/io/blockv/face/client/FaceManager.kt

```diff
@@ -78,7 +83,37 @@ interface FaceManager {
 
   val faceRegistry: Map<String, ViewFactory>
```

Comment:
On iOS, registry is called a roster. Should we align language?

Reply:
Will align in the next pr